ldes-server-port-admin-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/converters/ViewSpecificationConverter.java

```diff
@@ -0,0 +1,148 @@
+	}
+
+	private static List<Statement> retentionStatementsFromList(Resource viewName, List<RetentionConfig> retentionList) {
+		List<Statement> statements = new ArrayList<>();
+		for (RetentionConfig retention : retentionList) {
+			Resource retentionResource = createResource();
+			statements.add(createStatement(
+					retentionResource, createProperty(TYPE_PREDICATE), createResource(RETENTION_TYPE)));
+			retention.getConfig().forEach((key, value) -> statements.add(createStatement(
+					retentionResource, createProperty(key), createPlainLiteral(value))));
+			statements.add(createStatement(viewName, createProperty(RETENTION_OBJECT), retentionResource));
+		}
+		return statements;
+	}
+
+	private static List<FragmentationConfig> fragmentationListFromStatements(List<Statement> statements) {
+		List<FragmentationConfig> fragmentationList = new ArrayList<>();
+		for (Resource fragmentation : statements.stream().filter(new ConfigFilterPredicate(FRAGMENTATION_TYPE))
+				.map(Statement::getSubject).toList()) {
+			List<Statement> fragmentationStatements = retrieveAllStatements(fragmentation, statements);
+			FragmentationConfig config = new FragmentationConfig();
+			config.setName(fragmentation.toString());
+			config.setConfig(extractConfigMap(fragmentationStatements));
+			fragmentationList.add(config);
+		}
+
+		return fragmentationList;
+	}
+
+	private static List<Statement> fragmentationStatementsFromList(Resource viewName,
+			List<FragmentationConfig> fragmentationList) {
+		List<Statement> statements = new ArrayList<>();
+		for (FragmentationConfig fragmentation : fragmentationList) {
+			Resource fragmentationResource = createResource();
+			statements.add(createStatement(
+					fragmentationResource, createProperty(TYPE_PREDICATE), createResource(FRAGMENTATION_TYPE)));
+			fragmentation.getConfig().forEach((key, value) -> statements.add(createStatement(
+					fragmentationResource, createProperty(key), createPlainLiteral(value))));
+			statements.add(createStatement(viewName, createProperty(FRAGMENTATION_OBJECT), fragmentationResource));
+		}
+		return statements;
+	}
+
+	private static List<Statement> retrieveAllStatements(Resource resource, List<Statement> statements) {
+		List<Statement> statementList = new ArrayList<>();
+		statements.stream().filter(statement -> statement.getSubject().equals(resource))
+				.forEach(statement -> {
+					statementList.add(statement);
+					if (statement.getObject().isResource()) {
+						statementList.addAll(retrieveAllStatements(statement.getResource(), statements));
+					}
+				});
+		return statementList;
+	}
+
+	private static Map<String, String> extractConfigMap(List<Statement> statementList) {
+		Map<String, String> configMap = new HashMap<>();
+		statementList.stream().filter(statement -> !statement.getPredicate().toString().equals(TYPE_PREDICATE))
+				.forEach(statement -> configMap.put(statement.getPredicate().toString(),
+						statement.getObject().asLiteral().getString()));
```

Comment:
would propose for readability, and as a general convention, to split stream operation on a separate line each. 
so .stream() BR .filter(...) BR .foreach()